test/runtests.jl

```diff
@@ -20,7 +20,29 @@ cd(dirname(@__FILE__)) do
 
     @everywhere include("testdefs.jl")
 
-    reduce(propagate_errors, nothing, pmap(runtests, tests; err_retry=false, err_stop=true))
+    const max_worker_rss = parse(Int, get(ENV, "JULIA_TEST_MAXRSS_MB", "200"))
```

Comment:
Can we not have any limit by default? AFAIK we only have this problem on 64bit Linux Travis so far and it should be pretty easy to set a limit specific to travis. Having a default limit will make it harder to debug the test (given that it's already hard enough to attach to worker process).

Also, how does this interact with single process testing?